README.md

```diff
@@ -360,7 +360,47 @@ defaultProps = {
 `url` is the site you want to load in the embedded iframe, and `height` is the number representing the height of
 the iframe.
 
+### Using the image proptype
 
+The `image` propType will launch an image picker in Site Designer when editing the block config. You must specify a block config like this one:
+
+```
+configSpec = {
+    imageConfig: ElementPropTypes.image({
+        uriBase: ElementPropTypes.string,
+        imagePath: ElementPropTypes.string,
+        altText: ElementPropTypes.string
+        width: ElementPropTypes.number
+        height: ElementPropTypes.number
+    })
```

Comment:
@dan-grayson As we are not showing this fields in the SD, but just a "Pick media" button, the user does not need to specify the full shape in the prop. In contrast, the `embeddable` prop, also defined here, does need to specify the prop shape, as we need to generate a form for it.

Additionally, can you add a note for the product and category prop?